pkg/app/app.go

```diff
@@ -37,6 +44,12 @@ func Route(path string, n UI) {
 	routes[path] = n
 }
 
+// RouteRe binds the regular expression pattern to the given UI node.
+func RouteRe(pattern string, n UI) {
+	re := regexp.MustCompilePOSIX(pattern)
```

Comment:
You can inline this in the statement below:

```go
routesWithRegexp = append(routesWithRegexp, regexpRoute{
    regexp: regexp.MustCompilePOSIX(pattern),
    node:   n,
})
```

It avoids creating an intermediary variable and we can read how the node and the regexp are mapping to the regexpRoute without jumping to the definition.

Reply:
good suggestion. will do

Reply:
yes